github_repo_loc_analyser/code_analyzer.py

```diff
@@ -0,0 +1,49 @@
+import os
+import subprocess
+from datetime import datetime
+
+import git
+
+from github_repo_loc_analyser.github_api_querier import PossibleRepo
```

Comment:
That is kind of problematic, because we will get a circular import once the Analyzer needs to be created by my part of the code.

Maybe add a new datatype for what information the Analyzer needs here and I will create a instance of it before initiating the Analayzer. Or simply use more parameters for __init__ each with a primitive type.

Reply:
Will do